events.ical

```diff
@@ -0,0 +1,26 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:-//Bitcoin Design Community//ical-generator//EN
+NAME:Bitcoin Design Community Calls & Events
+X-WR-CALNAME:Bitcoin Design Community Calls & Events
+TIMEZONE-ID:US/Pacific
+X-WR-TIMEZONE:US/Pacific
+BEGIN:VEVENT
+UID:714652370@bitcoindesigners.org
+SEQUENCE:0
+DTSTAMP:20201014T160000Z
+DTSTART:20201014T160000Z
+DTEND:20201014T170000Z
+SUMMARY:Community Call #5
+URL;VALUE=URI:https://api.github.com/repos/BitcoinDesign/Meta/issues/27

```

Comment:
URL is to json format of the issue not the GH website

```suggestion
URL;VALUE=URI:https://github.com/repos/BitcoinDesign/Meta/issues/27
```

Reply:
Good catch. I just fixed the action to use the proper HTML URL instead of the API URL.